modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -203,6 +203,9 @@ public static class Client {
 
         /** Client handshake header error. */
         public static final int HANDSHAKE_HEADER_ERR = CLIENT_ERR_GROUP.registerErrorCode((short) 10);
+
+        /** Specified node is not found in the cluster. */
+        public static final int NODE_NOT_FOUND_ERR = CLIENT_ERR_GROUP.registerErrorCode((short) 11);
```

Comment:
This error is not specific to the client. The same issue can occur when using the embedded API, right?

Reply:
The API (interface) is the same for client and embedded. Exceptions and error codes must be the same.

Reply:
Ok, let's handle this separately: https://issues.apache.org/jira/browse/IGNITE-21390 (I've checked what happens in embedded mode, and it does not look good at all).

Here let's revert related changes (remove error code, keep existing exception with deprecated ctor).